apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000007_add_model_to_spans.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+--changeset BorisTkachenko:add_model_to_spans
+
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.spans
+    ADD COLUMN IF NOT EXISTS model String DEFAULT '',
+    ADD COLUMN IF NOT EXISTS provider String DEFAULT '',
+    ADD COLUMN IF NOT EXISTS total_estimated_cost Decimal32(7) DEFAULT toDecimal32(0, 7),
```

Comment:
Zero is already the default for numerical no-nullable fields, but we can confirm

Reply:
Yes, double checked.